pkg/scheduler/podtask/pod_task.go

```diff
@@ -258,14 +303,15 @@ func RecoverFrom(pod api.Pod) (*T, bool, error) {
 		case annotation.BindingHostKey:
 			hostname = v
 		case annotation.SlaveIdKey:
-			t.TaskInfo.SlaveId = mutil.NewSlaveID(v)
+			t.Spec.SlaveID = v
 		case annotation.OfferIdKey:
 			offerId = v
 		case annotation.TaskIdKey:
 			t.ID = v
-			t.TaskInfo.TaskId = mutil.NewTaskID(v)
 		case annotation.ExecutorIdKey:
-			t.TaskInfo.Executor = &mesos.ExecutorInfo{ExecutorId: mutil.NewExecutorID(v)}
+			// this is nowhere near sufficient to re-launch a task, but we really just
+			// want this for tracking
+			t.executor = &mesos.ExecutorInfo{ExecutorId: mutil.NewExecutorID(v)}
```

Comment:
should we set `ExecutorInfo.Source` to something like `"recovery"` here, to be able to distinguish at some later point where this object came from?